src/accounting.rs

```diff
@@ -35,7 +35,7 @@ pub fn get_http_request_cost(
     json_rpc_payload: &str,
     max_response_bytes: u64,
 ) -> u128 {
-    let nodes_in_subnet = METADATA.with(|m| m.borrow().get().nodes_in_subnet);
+    let nodes_in_subnet = TRANSIENT_SUBNET_SIZE.with(|m| *m.borrow());
```

Comment:
What's a "transient subnet size"?

Reply:
Data stored in main memory (rather than stable memory) uses a `TRANSIENT_` prefix. Open to suggestions for a different naming convention.